test/jtreg-ext/requires/VMProps.java

```diff
@@ -419,13 +419,15 @@ protected String vmHasDTrace() {
     }
 
     /**
-     * @return true if compiler in use supports RTM and false otherwise.
+     * @return "true" if compiler in use supports RTM and false otherwise.
```

Comment:
Why did you put true in quotes?? The false is not in quotes. If anything it should be `{@code true}` and `{@code false}` but as we are never going to generate javadoc for this the formatting is immaterial.

Reply:
I got the formatting for the comment from which ever existing comment that I copied
to create the new ones. For whatever reason, these functions return String so "true"
and "false" are accurate. I'll fix my new ones to all be the same, but I'm not going to
try to clean up the existing ones.

Reply:
Let me try to be more clear.

I updated the existing entry to `"true"` because the new functions that I added used `"true"`
because I copied that comment from an existing function. These functions return String so
I'm going to change the `false` to `"false"`.